projects/plugins/jetpack/modules/sharedaddy/sharing-service.php

```diff
@@ -696,13 +696,13 @@ public function __construct( $id, $total ) {
 	 * @param object $a Sharing_Service_Total object.
 	 * @param object $b Sharing_Service_Total object.
 	 *
-	 * @return bool
+	 * @return int -1, 0, or 1 if $a is <, =, or > $b
```

Comment:
The situation with these was weird. Returning `$a < $b` means `usort()` sees 0 or 1, resulting in a reverse-order sort.